pinot-filesystem/src/test/java/com/linkedin/pinot/filesystem/LocalPinotFSTest.java

```diff
@@ -99,7 +99,8 @@ public void testFS() throws Exception {
 
     File firstTempDir = new File(_absoluteTmpDirPath, "firstTempDir");
     File secondTempDir = new File(_absoluteTmpDirPath, "secondTempDir");
-    Assert.assertTrue(firstTempDir.mkdirs(), "Could not make directory " + firstTempDir.getPath());
+    localPinotFS.mkdir(firstTempDir.toURI());
```

Comment:
Does this test leave the temp directories around after it is done? It should clean them. And that requires a new api in the interface (rmdir).

Reply:
no, it cleans all tempdirs, i do have delete in pinotfs to support this

Reply:
Oh I see, delete is for file as well as directory I guess.